### DISCUSSION TITLE
en/chapter_preface/about_the_book/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1056] [Announcements]
# en/chapter_preface/about_the_book/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_preface/about_the_book/

**Comment by cyb3ralbert** [comment]
wow! it seems interesting! cheers.

**Comment by Mimi120** [comment]
Thank you to everyone who contributed to this book. I hope to gain / build my knowledge of data structures and algorithms.
